src/sentry/tasks/derive_code_mappings.py

```diff
@@ -34,7 +34,11 @@ def derive_missing_codemappings(dry_run=False) -> None:
     """
     Queue tasks to process each organization to derive missing codemappings.
     """
-    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
+    organizations = Organization.objects.filter(
+        status=OrganizationStatus.ACTIVE,
+        # Temporary change in order to enable the task until we fix perf issues
+        name__in=["sentry-test", "Sentry", "sentry", "Foo_org"],
```

Comment:
Two of these are for the test. The other two are to hit the Sentry org. I'm just uncertain if it's uppercase or lowercase.

Reply:
Can we use the feature flag for this?

Reply:
We could but did not want to open a 2nd PR on getsentry just for this since it is temporary.